docs/snapcraft-parts.md

```diff
@@ -0,0 +1,126 @@
+# Snapcraft parts
+
+Parts are the main building block to create snaps using Snapcraft. Parts have
+their own private space and lifecycle. Each part uses a `plugin`, which tells
+the part how to behave and what to do with the information inside it.
+
+As seen in the [article about snapcraft.yaml syntax](snapcraft-syntax.md)
+parts have general keywords that apply to all of them. In one case, you may
+want to enhance your part's functionality using `stage-packages` which end up
+bringing Ubuntu deb-based packages into your part, `filesets` to declare
+inclusion and exclusion sets, `organize` so make the artifact output for your
+part neater, `stage` and `snap` to make certain only the right set of files is
+seen at each stage (making use of `filesets` or not). An example integrating
+these concepts for a part called `example-part` using a hypothetical plugin
+called `null` would look like:
+
+	parts:
+	  example-part:
+	    type: null
+	    stage-packages:
+	      - gpg
+	      - wget
+	    organize:
+	      opt/bin: bin
+	    filesets:
+	      binaries:
+	       - bin/*
+	       - usr/bin/*
+	      headers:
+	       - *.h
+	       - -include
+	    stage:
+	      - $binaries
+	      - test/bin/test_app
+	      - $headers
+	    snap:
+	      - $binaries
+ 
+In this example, imagine that the `null` plugin actually builds something in
+its private *build* location using its private *source* directory as a base,
+and that it installs the usual set of files to the file, except it will 
+*install* the part's private install directory.
```

Comment:
what about: it will install the part to its private install directory.
I'm not sure if that's what you meant, though.